src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -711,6 +715,24 @@ private void DowngradeLockFileIfNeeded(LockFile lockFile)
             }
         }
 
+        private void TouchMetadata(CacheFile cacheFile)
+        {
+            foreach (var packageFilePath in cacheFile.ExpectedPackageFilePaths)
+            {
+                if(!PathUtility.IsChildOfDirectory(_request.DependencyProviders.GlobalPackages.RepositoryRoot, packageFilePath)) continue;
+
+                var nupkgMetadataPath = Path.Combine(Path.GetDirectoryName(packageFilePath), ".nupkg.metadata");
+                try
+                {
+                    File.SetLastAccessTimeUtc(nupkgMetadataPath, DateTime.UtcNow);
+                }
+                catch (Exception ex)
+                {
+                    RestoreLogMessage.CreateError(NuGetLogCode.NU1110, $"Touching the metadata file {nupkgMetadataPath} failed with {ex.Message}");
```

Comment:
User facing messages must come from string resources, so they can be localized into multiple languages.

Reply:
Also, something like `Updating last access time on file {0} failed with {1}` is more understandable, and less creepy, than "touching" for people who don't have knowledge of the unix `touch` command.

Reply:
Indeed, you're right. Will make that change.